test/boost/sstable_test.cc

```diff
@@ -813,3 +813,12 @@ BOOST_AUTO_TEST_CASE(test_empty_key_view_comparison) {
     BOOST_CHECK_EQUAL(0, lf.size());
     BOOST_CHECK(lf.begin() == lf.end());
 }
+
+SEASTAR_TEST_CASE(test_sstable_formatter) {
+    return test_using_reusable_sst(uncompressed_schema(), uncompressed_dir(), 1, [] (auto& env, shared_sstable sstp) {
+        BOOST_REQUIRE_EQUAL(fmt::format("{}", *sstp), uncompressed_dir() + "/la-1-big-Data.db");
```

Comment:
nit, might want to use `BOOST_CHECK_EQUAL()` here, as we might want to continue with other validations even if this one fails. the rationales behind this suggestion is, for instance, if `{:T}` does not format as expected, we still want to know if `{:I}` formats. but "BOOST_REQUIRE_*` just aborts the whole test case.

Reply:
Good point